JSDemos/Demos/CheckBox/Overview/Angular/description.md

```diff
@@ -0,0 +1,9 @@
+When you add a CheckBox to an application, set its [value](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#value) property. The CheckBox can have the following states: 
+
+- Checked (**value** is **true**).
+- Unchecked (**value** is **false**).
+- Undetermined (**value** is **undefined**).
+
+To handle value change, use two-way binding: bind the value property to a component property. In this demo, the value properties of the "Handle value change" and "Disabled" CheckBox components are bound to the same component property. Click the first CheckBox to see how it affects the second one. You can also use the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle value change, for example, if you need to use the previous value.
```

Comment:
```suggestion
To handle value changes, use a two-way binding: bind the value property to a component property. In this demo, the value properties of the "Handle value change" and "Disabled" CheckBox components are bound to the same component property. Click the first CheckBox to see how it affects the second one. You can also use the [onValueChanged](/Documentation/ApiReference/UI_Components/dxCheckBox/Configuration/#onValueChanged) property to handle value changes, for example, if you need to use the previous value.
```

Reply:
`use a two-way binding: bind the value property to a component property` Why do you use a colon here? Why do you write the sentence this way?

Reply:
I used the colon to avoid using the "using" in a sentence.